plonetheme/barceloneta/theme/less/barceloneta-compiled.css

```diff
@@ -2270,6 +2274,7 @@ fieldset[disabled] .link-parent.active {
   color: #777777;
 }
 .pagination {
+  display: inline-block;
```

Comment:
It's compiling from [pagination.plone.less, line 4](https://github.com/plone/plonetheme.barceloneta/blob/5633a99349a86a139e28cb58cb0422e357cd4bff/plonetheme/barceloneta/theme/less/pagination.plone.less). I don't know why it wasn't in [barceloneta-compiled.css](https://github.com/plone/plonetheme.barceloneta/blob/5633a99349a86a139e28cb58cb0422e357cd4bff/plonetheme/barceloneta/theme/less/barceloneta-compiled.css) already.